File: Selenium/Project/CRM_Activity1.java
package demosuit;

import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;
import org.testng.Assert;
import org.testng.annotations.AfterClass;
import org.testng.annotations.BeforeClass;
import org.testng.annotations.Test;

public class CRM_Activity1 {
	WebDriver driver;

	@Test
	// Setup function
	@BeforeClass
	public void setUp() {
		// Initialize Chrome driver
		driver = new ChromeDriver();
		// Open the page
		driver.get("https://alchemy.hguy.co/crm/");
	}
	 @Test
	    public void testCase1() {
	        // This test case will pass
	        String title = driver.getTitle();
	        System.out.println("Title is: " + title);
	        Assert.assertEquals(title, "SuiteCRM");
	    }


//Tear down function
	@AfterClass
	public void tearDown() {
		// Close the browser
		driver.quit();
	}
}

File: Selenium/Project/CRMActivity1.py
# Import webdriver from selenium
from selenium import webdriver
from selenium.webdriver.common.by import By

# Start the Driver
with webdriver.Chrome() as driver:
    # Navigate to the URL
    driver.get("http://alchemy.hguy.co/crm")

    # Print the title of the page
    title= driver.title
if title == "SuiteCRM":
        print("Title matches: SuiteCRM")
else:
        print(f"Title does not match. Found: {title}")
        driver.quit()